Client/TeamTalkAndroid/src/main/java/dk/bearware/gui/MainActivity.java

```diff
@@ -173,6 +173,7 @@ public class MainActivity
     TabLayout mTabLayout;
 
     public static final String TAG = "bearware";
+    public static List<Integer> userIDS = new ArrayList<>();
```

Comment:
Do not make this variable `static`. In `OnlineUsersActivity` you should call the `TeamTalkService.getUsers()` instead